ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/eea/EeaSendRawTransaction.java

```diff
@@ -77,87 +72,100 @@ public String getName() {
 
   @Override
   public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
-    final String rawPrivateTransaction = requestContext.getRequiredParameter(0, String.class);
     final Object id = requestContext.getRequest().getId();
+    final String rawPrivateTransaction = requestContext.getRequiredParameter(0, String.class);
 
     try {
       final PrivateTransaction privateTransaction =
           PrivateTransaction.readFrom(RLP.input(Bytes.fromHexString(rawPrivateTransaction)));
 
       final String enclavePublicKey =
           enclavePublicKeyProvider.getEnclaveKey(requestContext.getUser());
-      final ValidationResult<TransactionInvalidReason> validationResult =
-          privacyController.validatePrivateTransaction(privateTransaction, enclavePublicKey);
-      if (!validationResult.isValid()) {
-        return new JsonRpcErrorResponse(
-            id, convertTransactionInvalidReason(validationResult.getInvalidReason()));
+
+      if (!privateTransaction.getPrivateFrom().equals(Bytes.fromBase64String(enclavePublicKey))) {
```

Comment:
Can `privateTransaction.getPrivateFrom()` be null? Do we need to be careful here?

Reply:
I think the PrivateTransaction.readFrom does not return null ..